### PR TITLE
PIP-884-autogenerate-conf

### PR DESCRIPTION
**Pull request by paul-sud**

### CODE REVIEW

File: tests/test_make_conf.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                (12, "underconversion_sequence = chrL"),
+                (13, "include include.conf"),
+            ],
+        ),
+    ],
+)
+def test_make_conf(args, expected_num_rows: int, assertions: List[Tuple[int, str]]):
+    output = make_conf(args)
+    assert len(output) == expected_num_rows
+    for row_index, expected_value in assertions:
```
Comment (ottojolanki) [inline tests/test_make_conf.py:105]: I am too old to see world as black and white, and "one assertion per test", but is there a chance here that this test failing is uninformative?

**Review by ottojolanki — COMMENTED** [review]
If you are confident that you are not going to get into some weird assertion roulette with that one test, LGTM.

File: tests/test_make_conf.py (hunk 1 above)
Comment (paul-sud) [reply tests/test_make_conf.py:105]: It's unlikely. There are a few edge cases in `make_conf()` that are only tested here. Particularly here, I want to make sure that I test it when one of, none of, and both of `args.underconversion_sequence` and `args.include_file` are `None`. I just didn't want to assert the whole list is equal, since it is pretty redundant (most items in the list are constant).
